alacritty/src/window.rs

```diff
@@ -481,3 +490,17 @@ unsafe extern "C" fn xembed_error_handler(_: *mut XDisplay, _: *mut XErrorEvent)
     error!("Could not embed into specified window.");
     std::process::exit(1);
 }
+
+#[inline]
+#[cfg(not(any(target_os = "macos", windows)))]
+fn gl_clear(color: Rgb, alpha: f32) {
+    unsafe {
+        gl::ClearColor(
+            (f32::from(color.r) / 255.0).min(1.0) * alpha,
+            (f32::from(color.g) / 255.0).min(1.0) * alpha,
+            (f32::from(color.b) / 255.0).min(1.0) * alpha,
+            alpha,
+        );
+        gl::Clear(gl::COLOR_BUFFER_BIT);
+    }
+}
```

Comment:
We should never just copy-paste functions like this. If the old API doesn't work, it needs to be changed, not duplicated.

Reply:
I agree, you can do that with old API just fine.